math_keyboard/lib/src/foundation/node.dart

```diff
@@ -215,7 +212,7 @@ class Cursor extends TeX {
     }
     final colorString =
         '#${(cursorColor.value & 0xFFFFFF).toRadixString(16).padLeft(6, '0')}';
-    return '\\textcolor{$colorString}{|}';
+    return '\\textcolor{$colorString}{\\cursor{}}';
```

Comment:
Do we need a group here for the cursor?

```suggestion
    return '\\textcolor{$colorString}{\\cursor}';
```

Otherwise, I'd suggest changing that requirement in `flutter_math_for` :+1:

Reply:
Right, that group is not really needed there – removed 👍